.env.dev

```diff
@@ -34,7 +34,5 @@ FRANCECONNECT_CLIENT_SECRET=
 FRANCECONNECT_URL=https://fcp.integ01.dev-franceconnect.fr
 FRANCECONNECT_REDIRECT_URI=http://localhost:3000/api/auth/france-connect/callback
 FRANCECONNECT_POST_LOGOUT_REDIRECT_URI=http://localhost:3000/api/auth/france-connect/logout-callback
-CRISP_TOKEN_ID=
-CRISP_TOKEN_KEY=
-CRISP_WEBSITE_ID="064fca1b-bdd6-4a81-af56-9f38e40953ad"
+NEXT_PUBLIC_CRISP_WEBSITE_ID=064fca1b-bdd6-4a81-af56-9f38e40953ad
```

Comment:
why use an env variable if you commit it in the dev ? Or is it a dev SITE_ID ?

Reply:
It is possible to have a different crisp website id in staging/dev and production.

At the moment we don't have, so this website id is the one from production and it is indeed possible to remove the env var.